pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -799,6 +827,141 @@ def read_train2_bams(filename):
     return train2_bams
 
 
+def read_santa_cruz_entries(filename, santa_cruz_freeze_entry_tsv):
+    santa_cruz_entries = {}
+
+    with open(filename, 'r') as f:
+        reader = csv.DictReader(f, delimiter='\t')
```

Comment:
Also, not sure why the TSV file is chosen here. Why not the JSONL file which is much easier to parse, the code would be much simpler. No need so many if statements below